Sources/Logging/Logging.swift

```diff
@@ -260,3 +299,55 @@ internal final class StdoutLogger: LogHandler {
         }?.0 ?? "\(timestamp)"
     }
 }
+
+extension Logging.Metadata.Value: ExpressibleByStringLiteral {
+    public typealias StringLiteralType = String
```

Comment:
indent

Reply:
@tomerd what's wrong with the indent here? I had a bug in github yesterday where some indent was missing but that looks okay to me: 
<img width="631" alt="screenshot 2019-01-08 at 6 15 06 pm" src="https://user-images.githubusercontent.com/624238/50850285-546a5e00-1371-11e9-864e-b9e1bb5adb2e.png">